roles/openshift_health_checker/openshift_checks/efk_stack.py

```diff
@@ -0,0 +1,604 @@
+# pylint: disable=missing-docstring
+'''
+  Module for performing checks on an elasticseatch, fluentd, and kibana stack
```

Comment:
:+1: I have clearly been looking at this too long. I noticed that weeks ago and didn't fix it :)